lib/mix/lib/mix/tasks/release.ex

```diff
@@ -1017,8 +1017,15 @@ defmodule Mix.Tasks.Release do
     out_path = Path.join([release.path, "..", "..", tar_filename]) |> Path.expand()
     info(release, [:green, "* building ", :reset, out_path])
 
+    include_erts = not is_nil(release.erts_source)
+    erlang_libs = [:compiler, :kernel, :runtime_tools, :sasl, :stdlib]
```

Comment:
Instead of checking for specific erlang libs, we should just reject any application that does not have the according directory in "lib/#{name}-#{vsn}". If they are not there, then it means it came from erts. Same thing can be done by for erts. Check if the dir is there, if it isn't, do not add it.

Reply:
:+1: Will fix